### PR TITLE
runfix: Conversation read message and right-sidebar opening

### PR DESCRIPTION
**Pull request by przemvs**
- Fix for opening Right Sidebar on clikc invite pepole.
- Add animations while right sidebar is opened - right sidebar overlapping title-bar and input-bar
- Propably fix for readed messages, more details in comment in there (Conversation.tsx)

### CODE REVIEW

File: src/script/components/Conversation/Conversation.tsx [hunk 1]
```diff
@@ -325,7 +325,7 @@ const ConversationList: FC<ConversationListProps> = ({
     const isUnreadMessage = messageTimestamp > conversationEntity.last_read_timestamp();
     const isNotOwnMessage = !messageEntity.user().isMe;
 
-    let shouldSendReadReceipt = true;
+    let shouldSendReadReceipt = false;
```
Comment (przemvs) [inline src/script/components/Conversation/Conversation.tsx:328]: Restore previous default value from dev branch: https://github.com/wireapp/wire-webapp/blob/3dc266c3c1fb90597a6472aaef776b52248777fc/src/script/view_model/content/MessageListViewModel.ts#L271

Propably i forgot to change this value while testing.

File: src/script/components/MessagesList/Message/index.tsx [hunk 2]
```diff
@@ -73,7 +74,7 @@ export interface MessageParams extends MessageActions {
     deleteMessageEveryone: (conversation: Conversation, message: BaseMessage) => void;
   };
   messageRepository: MessageRepository;
-  onVisible?: () => void;
+  onVisible?: (() => void) | null;
```
Comment (PatrykBuniX) [inline src/script/components/MessagesList/Message/index.tsx]: Do we really need both `null` and `undefined`? Can't we stick to one falsy value? 😄

Comment (przemvs) [reply src/script/components/MessagesList/Message/index.tsx]: done :D

**Review by PatrykBuniX — APPROVED** [review]
lgtm

**Pull request merged**